torch/onnx/_internal/exporter/_dynamic_shapes.py

```diff
@@ -0,0 +1,372 @@
+"""Compatibility functions for the torch.onnx.export API."""
+
+# mypy: allow-untyped-defs
+from __future__ import annotations
+
+import inspect
+import re
+import warnings
+from typing import Any, Sequence
+
+import torch
+from torch.export.dynamic_shapes import _Dim, _DimHint
```

Comment:
Is it possible to import modules only?

Reply:
I don't want to use dynamic_shapes._Dim, so I didn't import modules. The module name dynamic_shapes is too general. Do you think 

```python
from torch.export import dynamic_shapes as export_dynamic  # or some other recommendations?
```

is better than importing _Dim and _DimHint?
I don't have preference, just didn't like dynamic_shapes

Reply:
Feel free to keep the original, thanks!